lib/membrane_rtc_engine/endpoints/webrtc/track_receiver.ex

```diff
@@ -127,13 +125,22 @@ defmodule Membrane.RTC.Engine.Endpoint.WebRTC.TrackReceiver do
         else: state.forwarder
 
     {forwarder, buffer} = Forwarder.align(forwarder, buffer)
-    actions = if buffer, do: [buffer: {:output, buffer}], else: []
+
+    actions =
+      if buffer do
+        # if state.track.type == :video do
+        #   IO.inspect(buffer.metadata.rtp.sequence_number, label: "#{inspect(self())}")
+        # end
+
+        [buffer: {:output, buffer}]
+      else
+        []
+      end
```

Comment:
Welp, that's my leftover :sweat_smile: 

```suggestion
    actions = if buffer, do: [buffer: {:output, buffer}], else: []
```